client/src/components/Footer.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "../styles/Footer.css";
+import CYFLogo from "../assets/CYF.jpeg";
+import { Divider, Grid, Image, Segment } from "semantic-ui-react";
+
+const Footer = () => {
+  return (
+    <div className="footer">
+      <Grid divided columns={3}>
+        <Grid.Column>
+          {" "}
+          <div className="team-container">
+            <p>Min-Imece-Team</p>
+            <span>Samuel</span>
+            <span>Hacer</span>
+            <span>Fatma</span>
+            <span>Mawaddah</span>
+            <span>Natalia</span>
+          </div>
+        </Grid.Column>
+        <Grid.Column>
+          <h3>
+            Migrants' Questions and Answers is an open source project, created
```

Comment:
The Trello card says we should use the following text for this section:
```
A dashboard for Maryhill Integration Network that staff can use to better track, and support service users and volunteers created by CodeYourFuture Students
```
Let's update it to the text that the Trello card says